cmake/vt_add_test.cmake

```diff
@@ -64,11 +64,15 @@ function(vt_add_test)
         CXX_EXTENSIONS NO
     )
 
-    add_test(
-        NAME ${VALUE_TYPES_NAME}
-        COMMAND ${VALUE_TYPES_NAME}
-        WORKING_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
-    )
+    if(${VALUE_TYPES_MANUAL})
+        message(STATUS "Manual test: ${VALUE_TYPES_NAME}")
+    else()
+        add_test(
+            NAME ${VALUE_TYPES_NAME}
+            COMMAND ${VALUE_TYPES_NAME}
+            WORKING_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
+        )
+    endif()
 
     include(GoogleTest)
     gtest_discover_tests(${VALUE_TYPES_NAME})
```

Comment:
I don't this it make sense to call `gtest_discover_tests` it you have not called the branch that calls `add_test`

Reply:
Fixed.